src/lib.rs

```diff
@@ -37,6 +38,17 @@
 // This one is hard to avoid.
 #![allow(clippy::multiple_crate_versions)]
 
+#[allow(unused)]
```

Comment:
I am wondering if `format_error` and `verbose_logs` are the best names. The purpose of this flag is to hide the error messages coming from the crypto APIs that we use, unless you want to use it for other things?
If it is a security option, maybe it could be a bit more obvious in its name:
* `verbose_log` -> `print_crypto_lib_errors`
* `format_error` -> `crypto_lib_error`

Reply:
I think a change in `verbose_log` makes sense, but `format_error` seems reasonable, since it handles the whole formatting decision for the caller.
Also, it's not just about crypto library errors, it's also about errors coming from `bincode` when (de)serializing `KeyInfo` structures, for example.

`verbose_log` -> `log_error_objects`? `log_error_details`?

Reply:
`log_error_details` seems good!